### ISSUE TITLE
CLONE with recursive:true behaviour

### ISSUE THREAD

**Issue by Ingegneus** [ArnoldSmith86/virtualtabletop#1662]
atm when using CLONE with offsetX/Y or properties and recursive:true only changes those properties for the root widget. how do you guys feel about it? should it maybe affect all recursevly cloned widgets?

**Comment by flackr** [comment]
I don't think it should affect recursively cloned widgets. For my use case for this the top level item is special, may need different property values, but the descendants are just components of that widget. I think that changing all of the descendants would make it unusable for complex widgets and I'd just have to go back to manually implementing a recursive clone if we changed this to affect the descendants.

**Comment by flackr** [comment]
E.g. if i clone a player board which has a few holders I don't want to move those holders on the cloned player boards, nor do i want to apply properties that make sense on the root widget to all of the cloned descendants.
